docs/locations-and-scheduling.md

```diff
@@ -29,77 +29,106 @@ The APIs used for Compute as a Service are:
   Locations are visible to users, but owned by the compute service team, i.e. read-only to the users and only projected
   into their workspaces for visibility. A placement decision references a location by name.
 
+  `SyncTarget`s in a `Location` are transparent to the user. Workload should be able to seamless move from one `SyncTarget` to another
+  within a `Location`. It is compute service's responsibility to ensure that:
+
+  1. Network connectivity: a workload in one `SyncTarget` can access service in another `SyncTarget` with a predefined DNS format.
+  2. Storage compatibility: a storage in one `SyncTarget` can be easily moved to another `SyncTarget`.
+
+- `Placement` in `scheduling.kcp.dev/v1alpha1` – represents a selection rule to choose ONE `Location` via location labels, and bind
+  the selected location to MULTIPLE namespaces in a user workspace. Users can create multiple `Placement`s to select multiple `Locations`
```

Comment:
Is the point here to indicate that multiple namespaces can end up in different locations if desired? Eg, is this more accurate/clear:

> For Workspaces with multiple Namespaces, users can create multiple `Placement`s to assign specific Namespace(s) to specific `Locations`.